remote/Dockerfile

```diff
@@ -45,12 +46,10 @@ RUN wget -qO /usr/share/keyrings/xpra.asc ${XPRA_REGISTRY}/xpra.asc && \
         nginx && \
     rm -rf /var/lib/apt/lists/*
 
+COPY openbox-launcher.sh /usr/local/bin/openbox-launcher.sh
 COPY rc.xml /etc/xdg/openbox/rc.xml
 COPY menu.xml /etc/xdg/openbox/menu.xml
-
-# Setup Nitrogen (Desktop background)
-COPY wallpaper.png /tmp/wallpaper.png
-COPY bg-saved.cfg /home/techuser/.config/nitrogen/bg-saved.cfg
+RUN printf '\nGTK_IM_MODULE=ibus\n' >> /etc/environment
```

Comment:
![image](https://github.com/user-attachments/assets/3fe51a95-a8f1-4809-96cb-7253a197d1d9)

The corresponding environment variable isn't recognized by Capella. This leads to the old "flickering" / non-responsive behaviour in some cases.

Reply:
Fixed via `xpra [...] --input-method=keep`, which prevents it from setting GTK_IM_MODULE in the first place.